src/es/components/controllers/checkout/Checkout.js

```diff
@@ -58,6 +58,8 @@ export default class Checkout extends Shadow() {
         })
       }
 
+      const mitAnnulationskostenversicherung = this.querySelector('#mit-annulationskostenversicherung')?.checked
```

Comment:
Event Driven Rules:
- A controller must not query-select nor output any css or html. The communication must be strictly through events.

https://github.com/mits-gossau/web-components-toolbox/blob/master/README.md?plain=1#L25